components/modal-clubs-remove.jsx

```diff
@@ -0,0 +1,80 @@
+var React = require('react/addons');
```

Comment:
everything in this file was extracted from `pages/clubs.jsx`](https://github.com/mozilla/teach.webmaker.org/blob/develop/pages/clubs.jsx)